discuss/get_started.rst

```diff
@@ -0,0 +1,11 @@
+========
+Overview
```

Comment:
```suggestion
Get Started
```

Reply:
Unresolved

Reply:
I want to keep the name title as the one I chose. Thanks!

Reply:
I understand but it still looks, and will continue looking, like a mistake. You have a file named `get_started` whose title is "Overview" and a file named `overview` whose title is "Get started".

Maybe swapping file names or finding with new titles/structure could solve this ?

Reply:
Sure